src/index.html

```diff
@@ -2,16 +2,155 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="container">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text" name="Surname"
+            required autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            min="1980-01-01"
+            max="2022-01-01"
+          >
+        </label>
+
+        <label class="form-field position-bottom">
+          I accept the term of the agreement
+          <input type="checkbox" name="termOfAgreement">
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label  class="form-field position-bottom">
+          Password:
+          <input
+            type="password"
+            name="pass"
+            minlength="1"
+            maxlength="20"
+          >
+
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <div  class="form-field">
+          <label>
+            Do you love cats?
+            <input
```

Comment:
Move this text out of label, label is connected to radio option
```suggestion
        <div class="form-field">
          Do you love cats?
          <label>
            <input
```